docs/reference/cluster/update-desired-nodes.asciidoc

```diff
@@ -92,7 +100,8 @@ The API returns the following result:
 [source,console-result]
 --------------------------------------------------
 {
-  "replaced_existing_history_id": false
+  "replaced_existing_history_id": false,
+  "dry_run": false
```

Comment:
I'm not entirely convinced about the need of returning this in the response, the caller should know that this was invoked with dry run?

Reply:
I wanted add it for the consistency stake. There are a few APIs that offer the dry run option and they return the `dry_run` field.